android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -594,4 +595,11 @@ public String getApiName()
 	{
 		return "Ti.UI.Window";
 	}
+
+	private void logDeprecatedPropertiesWarning() {
```

Comment:
You should just call `Log.w` where you need to, no need to create another method.